### PR TITLE
mount: add --bind example

### PR DESCRIPTION
**Pull request by cpriest**
- [ ] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/mount.md [hunk 1]
```diff
@@ -21,3 +21,7 @@
 - Mount a specific filesystem described in /etc/fstab (e.g. "/dev/sda1 /my_drive ext2 defaults 0 2"):
 
 `mount {{/my_drive}}`
+
+- Mount a directory to another directory (bind mount) \{ --bind || -B \} {{olddir}} {{newdir}}:
+
+`mount --bind {{/tmp}} {{/usr/tmp}}`
```
Comment (agnivade) [inline pages/common/mount.md]: I think you can write placeholder dirs here itself to make it simple and clear. Concrete examples here does not make it clear which is the olddir and which is the newdir. Please write it in the form of {{path/to/old_dir}}.

And we can also remove "(bind mount) \{ --bind || -B \}"

Comment (cpriest) [reply pages/common/mount.md]: How's that look?

Comment (agnivade) [reply pages/common/mount.md]: The directory values should be path/to/old_dir and path/to/new_dir respectively

File: pages/common/mount.md [hunk 2]
```diff
@@ -21,3 +21,7 @@
 - Mount a specific filesystem described in /etc/fstab (e.g. "/dev/sda1 /my_drive ext2 defaults 0 2"):
 
 `mount {{/my_drive}}`
+
+- Mount a directory to another directory:
+
+`mount --bind {{olddir}} {{newdir}}`
```
Comment (mebeim) [inline pages/common/mount.md]:
```suggestion
`mount --bind {{path/to/old_dir}} {{path/to/new_dir}}`
```

**Pull request merged**